### PR TITLE
fix/return non-zero codes on errors

### LINKED ISSUE #657: [0.22.0 BUG]: CLI returns 0 exit-code on non-existent object get
## Expected Behavior
Non-zero exit-code.

## Current Behavior
```
$ neofs-cli --rpc-endpoint <rpc> --wif <wif> object get --cid <cid> --oid <oid> --file file
can't get object: reading the response failed: rpc error: code = Unknown desc = object already removed
$ echo $?
0
```

## Steps to Reproduce (for bugs)
1. put an object
2. delete that object
3. get tombstone
4. get the object via neofs-cli

## Regression
Yes. In previous version CLI returned non-zero ec.

## Your Environment
```
$ neofs-cli --version
Version: v0.22.0 
Build: 2021-06-30T16:38:25+00:00 
Debug: false
```
```
$ uname -a
Linux username 4.15.0-144-generic #148-Ubuntu SMP Sat May 8 02:33:43 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

### PR DESCRIPTION
**Pull request by carpawell**
Closes #657.

### CODE REVIEW

File: cmd/neofs-cli/modules/util.go [hunk 1]
```diff
@@ -508,3 +445,30 @@ func keyerParseFile(filename string, d *keyer.Dashboard) error {
...
+
+// exitOnErr prints error via cmd and exits with
+// "1" exit code. Prints and exits only on non-nil
+// errors.
+func exitOnErr(cmd *cobra.Command, err error) {
+	exitOnErrCode(cmd, err, 1)
+}
+
+// exitOnErr prints error via cmd and exits with
+// passed exit code. Prints and exits only on non-nil
```
Comment (cthulhu-rider) [inline cmd/neofs-cli/modules/util.go:467]:
```suggestion
// passed exit code. Does nothing if err is nil.
```

**Review by cthulhu-rider — APPROVED** [review]

**Review by alexvanin — APPROVED** [review]

File: cmd/neofs-cli/modules/util.go [hunk 2]
```diff
@@ -508,3 +445,30 @@ func keyerParseFile(filename string, d *keyer.Dashboard) error {
 
 	return d.ParseBinary(data)
 }
+
+// errf returns formatted error in errFmt format
+// if err is non-nil error.
```
[the remark this answers is not in the document]
Comment (carpawell) [reply cmd/neofs-cli/modules/util.go:450]: changed

File: cmd/neofs-cli/modules/util.go [hunk 3]
```diff
@@ -508,3 +445,30 @@ func keyerParseFile(filename string, d *keyer.Dashboard) error {
...
+func errf(errFmt string, err error) error {
+	if err == nil {
+		return nil
+	}
+
+	return fmt.Errorf(errFmt, err)
+}
+
+// exitOnErr prints error via cmd and exits with
```
[the remark this answers is not in the document]
Comment (carpawell) [reply cmd/neofs-cli/modules/util.go:459]: changed

File: cmd/neofs-cli/modules/util.go [hunk 4]
```diff
@@ -508,3 +445,30 @@ func keyerParseFile(filename string, d *keyer.Dashboard) error {
...
+// "1" exit code. Prints and exits only on non-nil
+// errors.
+func exitOnErr(cmd *cobra.Command, err error) {
+	exitOnErrCode(cmd, err, 1)
+}
+
+// exitOnErr prints error via cmd and exits with
```
[the remark this answers is not in the document]
Comment (carpawell) [reply cmd/neofs-cli/modules/util.go:466]: changed

File: cmd/neofs-cli/modules/util.go (hunk 1 above)
Comment (carpawell) [reply cmd/neofs-cli/modules/util.go:467]: changed

**Pull request merged**